src/androidTest/java/com/couchbase/lite/performance/Test3_CreateDocsWithAttachments.java

```diff
@@ -80,8 +73,23 @@ public boolean run() {
                 return true;
             }
         });
+        assertTrue(success);
 
-        Log.v("PerformanceStats", TAG + "," + Long.valueOf(System.currentTimeMillis() - startMillis).toString() + "," + getNumberOfDocuments() + "," + getSizeOfAttachment());
+        Log.e("PerformanceStats", TAG + "," + Long.valueOf(System.currentTimeMillis() - startMillis).toString() + "," + getNumberOfDocuments() + "," + getSizeOfAttachment());
+    }
+
+    private static Map<String, Map<String, Object>> getAttachmentsDict(byte[] data, String name, String type, boolean gzipped) {
+        if (gzipped)
+            // TODO
+            ;
```

Comment:
Current AttachmentTest does not support a gzipped attachment. So, `TODO` is for future task.